RubberduckBaseProject.csproj

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project>
```

Comment:
This is easily my favorite feature of the new csproj format -- ability to standardize all projects. Love that! 👍